src/Resource/Environment.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * This file is part of the contentful.php package.
+ *
+ * @copyright 2015-2018 Contentful GmbH
+ * @license   MIT
+ */
+
+namespace Contentful\Delivery\Resource;
+
+class Environment extends BaseResource
+{
+    /**
+     * @var Locale[]
+     */
+    protected $locales = [];
+
+    /**
+     * @return Locale[]
+     */
+    public function getLocales()
+    {
+        return $this->locales;
+    }
+
+    /**
+     * @param string $code Code of the locale to fetch the object for
+     *
+     * @throws \InvalidArgumentException When no locale with the given code exists
+     *
+     * @return Locale
+     */
+    public function getLocale($code)
```

Comment:
I personally don't think that this should be here, as it is a new endpoint.

Reply:
It's a special case... You don't have `/spaces/xxx/environments/yyy/locales/zzz`, you only have the collection endpoint.

To me it makes sense to keep it here, as it's consistent with the previous SDK behavior, and if you want to get locales you can simply do:

``` php
$locales = $client->getEnvironment()->getLocales();
```

It's functionally the same, with the advantage that with this approach, I can much more easily handle caching, because of inherent limitations of working with a collection endpoint rather than using a regular ID for querying.